docs/supported_platforms.md

```diff
@@ -233,3 +233,42 @@ The kubeconfig is generated by command:
 `ocm get /api/clusters_mgmt/v1/clusters/{cluster_id}/credentials`
 
 And stored in the same structure like openshift-installer does in cluster dir.
+
+## ROSA
+
+This platform supports deployment of OCP cluster + OCS cluster on top of it.
+
+### Requirements
+
+* Provide config file + credential config file described bellow to
+`run-ci` command via `--ocsci-conf` parameters.
+
+### Configuration files
+
+* `conf/ocsci/rosa.yaml` - this file can be used for deployment on ROSA.
+
+You will need to create also credential file with secret data which should
+never be shared publicly in this repository.
+All needed values are mentioned as placeholders and commented out in
+`conf/ocsci/rosa.yaml` and also mentioned below.
+
+```yaml
+# This is the basic config for OCM, user needs to login to OCM before ROSA can
```

Comment:
uncomment?

Reply:
I am not sure. Other platform descriptions in docs/supported_platforms.md  have comments like these before some config excerpt.